app/actions/user.ts

```diff
@@ -81,6 +85,19 @@ export const loadAccountDetails = (params: {
         IERC20.abi,
         params.provider
       );
+      // remember to add this to addresses object
+      const klimaDomainContract = new ethers.Contract(
+        "0xe8b97542A433e7eCc7bB791872af04DF02A1a6E4",
```

Comment:
pls export from the addresses object in @klimadao/lib/constants, leave empty string on testnet if none exists

Reply:
fixed